srcpkgs/libopensmtpd/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'libopensmtpd'
+pkgname=libopensmtpd
+version=0.7
+revision=1
+hostmakedepends="mdocml"
```

Comment:
This is just used to format and then compress the man page, `xbps-src` uncompresses it again.
If you set `make_build_args="MANFORMAT="` and `make_install_args="MANFORMAT="` it should work without that extra dependency and just install the uncompressed mandoc file.